### ISSUE TITLE
If a columns element is set to never stack, changing a setting via Advanced > Typography causes the column to stack at that break point

### ISSUE THREAD

**Issue by Ravensfoot** [soflyy/oxygen-bugs-and-features#1146]
Stack Columns Vertically set to "never" 
Set editing to "Less than 768px"
Select Column
Click Advanced -> Typography 
Change font size to 12 pt and the columns switch to "Stack Vertically" even though the setting remains set to "Never" .

![Screen Shot 2020-05-12 at 2 24 07 PM](https://user-images.githubusercontent.com/65256834/81747216-8ce5e480-945c-11ea-91e6-5a4153466006.png)
![Screen Shot 2020-05-12 at 2 24 23 PM](https://user-images.githubusercontent.com/65256834/81747222-8fe0d500-945c-11ea-9664-cd1832894293.png)
![Screen Shot 2020-05-12 at 2 24 50 PM](https://user-images.githubusercontent.com/65256834/81747225-91aa9880-945c-11ea-976c-227167fc16e0.png)

**Comment by Ravensfoot** [comment]
![Screen Shot 2020-05-12 at 2 24 50 PM](https://user-images.githubusercontent.com/65256834/81747154-73dd3380-945c-11ea-8add-ef6907cd8354.png)
![Screen Shot 2020-05-12 at 2 24 23 PM](https://user-images.githubusercontent.com/65256834/81747158-750e6080-945c-11ea-80c8-d04d8b7cb73b.png)
![Screen Shot 2020-05-12 at 2 24 07 PM](https://user-images.githubusercontent.com/65256834/81747159-750e6080-945c-11ea-9abc-53e71dafae33.png)

**Comment by katsar0v** [comment]
Is there an update @KittenCodes ?

**Comment by mousebat** [comment]
Still getting this issue - can someone fix it please!  I'm getting the bug with margin top as per the closed duplicate here:

https://github.com/soflyy/oxygen-bugs-and-features/issues/1824

**Comment by mousebat** [comment]
```diff
@@ -2735,18 +2735,6 @@ Class CT_Component {
 			}
 
 			$selector_css = "";
-			$default_stack_columns_vert = "";
-
-			// default 
-			if (!isset($atts['stack-columns-vertically']) && $this->options['tag'] == "ct_new_columns") {
-				$default_stack_columns_vert .= "@media (max-width: ".$media_queries_list[$default_atts['stack-columns-vertically']]['maxSize'].") {";
-				$default_stack_columns_vert .= '#' . $selector . "> .ct-div-block {width: 100% !important;}";
-				$default_stack_columns_vert .= "}";
-			}
-
-			if ( sizeof($atts) < 1 ) {
-				$styles .= $default_stack_columns_vert;
-			}
 
 			// handle units
 			foreach ( $atts as $param => $value ) {
@@ -2871,8 +2859,6 @@ Class CT_Component {
 					$stackColumnsVerticallyStyles .= "}";
 				}
 
-				$styles .= $default_stack_columns_vert;
-				
 				if ( isset($atts['reverse-column-order']) && 
 					 $atts['reverse-column-order'] != 'never' && 
 					 $atts['reverse-column-order'] != 'always' ) {
```
I've removed this as it seemed superfluous but I may be wrong.  The builder still displays incorrectly but at least the live styles aren't borked.

**Comment by KittenCodes** [comment]
Fixed in v3.8 Beta 1.

**Comment by swinggraphics** [comment]
@KittenCodes I hope 3.8 is coming soon. This is incredibly aggravating. It would be good if fixes showed up in more frequent minor updates. I really wish there were 3.7.2, 3.7.3… We spend an inordinate amount of time fighting with Oxygen to fix bad behavior.
